build/coreos-4/component.yaml

```diff
@@ -11974,13 +11974,13 @@ satisfies:
 - control_key: SI-14
```

Comment:
This change should be removed from your PR.

Reply:
Attempted rebase to remove the /build/* changes from the parent lineage of commits.